.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Does this need to be configured per-repo? Seems like a sensible default for a user config. Would prefer to minimize editor-specific config, but I use NeoVim so I don't really have a voice here.

Reply:
Yes, or every user has to have it in their user settings.